### PR TITLE
Add `.as_bytes()` method for `StaticSecret`

### PR DESCRIPTION
**Pull request by dlblv**

### CODE REVIEW

File: src/x25519.rs [hunk 1]
```diff
@@ -174,7 +174,7 @@ impl StaticSecret {
     /// Perform a Diffie-Hellman key agreement between `self` and
     /// `their_public` key to produce a `SharedSecret`.
     pub fn diffie_hellman(&self, their_public: &PublicKey) -> SharedSecret {
-        SharedSecret(&self.0 * their_public.0)
+        SharedSecret(self.0 * their_public.0)
```
Comment (tarcieri) [inline src/x25519.rs:177]: @dlblv did you benchmark this change?

File: tests/x25519_tests.rs [hunk 2]
```diff
@@ -1,4 +1,3 @@
-
```
Comment (tarcieri) [inline tests/x25519_tests.rs:1]: All of the changes to this file seem unrelated and should probably go in a separate rustfmt PR

File: src/x25519.rs (hunk 1 above)
Comment (dlblv) [reply src/x25519.rs:177]: @tarcieri yes, no performance losses

File: tests/x25519_tests.rs (hunk 2 above)
Comment (dlblv) [reply tests/x25519_tests.rs:1]: will revert them then

Comment (dlblv) [reply tests/x25519_tests.rs]: @tarcieri done

**Review by tarcieri — APPROVED** [review]
utACK

**Comment by dlblv** [comment]
@tarcieri what shall I done to make this PR be merged?
